app/policies/pdf_template_policy.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class PdfTemplatePolicy < ApplicationPolicy
+  def index?
+    user.program_admin? || user.staff?
+  end
+
+  def create?
+    index?
```

Comment:
@hengsokly please let the only program_admin? to create/update template as it's one for permernant